cmdstanpy/install_cmdstan.py

```diff
@@ -365,6 +363,9 @@ def compile_example(verbose: bool = False) -> None:
         # pylint: disable=raise-missing-from
         raise CmdStanInstallError(f'Command "make clean-all" failed\n{e}')
```

Comment:
I know this wasn't in your changes, but it looks like this message is wrong and should be updated while we're here